component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/twitter/TwitterAuthenticatorConstants.java

```diff
@@ -20,8 +20,8 @@
 package org.wso2.carbon.identity.authenticator.twitter;
 
 public class TwitterAuthenticatorConstants {
-    public static final String AUTHENTICATOR_NAME = "TwitterAuthenticator";
-    public static final String AUTHENTICATOR_FRIENDLY_NAME = "Twitter";
+    public static final String AUTHENTICATOR_NAME = "Twitter";
+    public static final String AUTHENTICATOR_FRIENDLY_NAME = "TwitterAuthenticator";
```

Comment:
The authenticator friendly name  should be  'Twitter' and authenticator name should be "TwitterAuthenticator" to be consistency with other authenticators.